data/brands/shop/alcohol.json

```diff
@@ -450,6 +450,25 @@
         "shop": "alcohol"
       }
     },
+    {
+      "displayName": "Liquor Mart",
```

Comment:
Would "Manitoba Liquor Mart" be more appropriate?

Reply:
Signage just says "Liquor Mart" as does website https://www.liquormarts.ca/

Example: https://www.openstreetmap.org/node/4927513512 Look at the Bing street view imagery. Sign is just Liquor Mart. Mapper mapped this as "Carman Liquor Mart" though.

Reply:
The website does use Manitoba Liquor Mart in some places, but I'll agree that Liquor Mart is used primarily in branding. I'll merge as-is.